hack/verify-deep-copies.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
+DESCHEDULER_ROOT=$(dirname "${BASH_SOURCE}")/..
+mkdir -p "${DESCHEDULER_ROOT}/_tmp"
+_tmpdir="$(mktemp -d "${DESCHEDULER_ROOT}/_tmp/kube-verify.XXXXXX")"
+
+_deschedulertmp="${_tmpdir}"
+mkdir -p "${_deschedulertmp}"
+
+git archive --format=tar --prefix=descheduler/ "$(git write-tree)" | (cd "${_deschedulertmp}" && tar xf -)
+_deschedulertmp="${_deschedulertmp}/descheduler"
+
+pushd "${_deschedulertmp}" > /dev/null 2>&1
+go build -o "${OS_OUTPUT_BINPATH}/deepcopy-gen" "k8s.io/code-generator/cmd/deepcopy-gen"
+
+${OS_OUTPUT_BINPATH}/deepcopy-gen \
+                --go-header-file "hack/boilerplate/boilerplate.go.txt" \
+                --input-dirs "./pkg/apis/componentconfig,./pkg/apis/componentconfig/v1alpha1,./pkg/api,./pkg/api/v1alpha1" \
+                --output-file-base zz_generated.deepcopy
+popd > /dev/null 2>&1
+
+pushd "${DESCHEDULER_ROOT}" > /dev/null 2>&1
+if ! _out="$(diff -Naupr pkg/ "${_deschedulertmp}/pkg/")"; then
+    echo "Generated deep-copies output differs:" >&2
+    echo "${_out}" >&2
+    echo "Generated deep-copies verify failed."
+    exit 1
+fi
+popd > /dev/null 2>&1
+
+echo "Generated deep-copies verified."
```

Comment:
Please delete this line. You can also remove this line from the verify-vonversions.sh script too.

Reply:
Are you referring to, line 34 or line 36 ?

Reply:
Line 36

Reply:
Some kind of "success" message is helpful than printing nothing, to show that the script did actually complete (for local runs, and for CI where we are calling multiple scripts).

I would also update the failure message to include a `Please run ./hack/update-deep-copies.sh` (same for conversions if it doesn't already)

Reply:
Updated with latest push.

Reply:
Okay, I'll just put back those in with the updated failure message as well.

Reply:
Yeah, sorry for the back and forth but I do think these are worth having... You may want to wait for @seanmalloy to see if he agrees or there is something I'm overlooking 🙂

Reply:
It's fine to keep it.

Reply:
@pravarag you can re-add the `echo` statement. Sorry for the confusion.